pkg/clusterregistryconfig/flags.go

```diff
@@ -147,12 +147,17 @@ func GetClusterRegistryConfigOptions(cmd *pflag.FlagSet,
 	}
 
 	if interactive.Enabled() && isHostedCP {
+		regConfigQuestion := "Enable registries config"
```

Comment:
Maybe extract the next few lines to a separate function so that we can add unit tests for it.

Reply:
Let me know if the changes I pushed are what you are expecting.